1.13/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM nginx:alpine
 
-WORKDIR /usr/share/nginx/html
-
 ENV OPENHIM_CONSOLE_VERSION 1.13.2
 
+WORKDIR /etc
 RUN wget -qO- "https://github.com/jembi/openhim-console/releases/download/v$OPENHIM_CONSOLE_VERSION/openhim-console-v$OPENHIM_CONSOLE_VERSION.tar.gz" | tar xvz
+
+WORKDIR /usr/share/nginx
+RUN rm -rf html
+RUN ln -s /etc/openhim-console/dist html
```

Comment:
@Zooloo2014 im not sure this is actually needed to be updated. We just need to ensure the zip file we upload to the tagged release is only the what is in the dist folder. Have a look at some of the previous releases to see how the contents of the zip looks

Reply:
:+1: